rbc_base:
  OmniwheelBaseController:
    type: "rbc_base/OmniwheelBaseController"
    joint_name_1: "wheel_1_joint"
    joint_name_2: "wheel_2_joint"
    joint_name_3: "wheel_3_joint"
    joint_name_4: "wheel_4_joint"
joint_state_controller:
  JointStateController:
    type: "joint_state_controller/JointStateController"
    publish_rate: 50